commands/moderation/ban.js

```diff
@@ -0,0 +1,70 @@
+const Discord = require('discord.js');
+var dateFormat = require('dateformat');
+
+module.exports = {
+    name: "ban",
+    description: "Ban a user",
+    
+    execute(msg, con, args) {        
+        if (!msg.member.roles.cache.some(
+            role => role.name === "Admin")) {
+                return msg.reply("You must be an Admin to use this command.");
+        }else{
+            const toBan = msg.mentions.members.first();
+            if (!toBan) {
+                return msg.reply("Please provide a user to ban.");
+            }
+
+            if(msg.mentions.members.first() == msg.author) {
+                return msg.reply("You can't ban yourself!")
+            }
+
+            if(toBan.hasPermission('BAN_MEMBERS')) {
+                return msg.reply("This user also has ban privileges.")
+            }
+
+            const reason = msg.content.substr(msg.content.indexOf(">") + 2);
+            if (reason === "") {
+                return msg.reply("Please provide a reason for banning.");
+            }
+            
+            let now = new Date();
+            let date = dateFormat(now, "yyyy-mm-dd HH:MM:ss");
+
+            const action = "cc!ban " + args.join(" ")
+
+            console.log(action)
+            // Inserts row into database
+            var sql = `INSERT INTO infractions (timestamp, user, action, length_of_time, reason, valid, moderator) VALUES 
+            ('${date}', '${toBan.id}', 'cc!ban', 'N/A', '${reason}', true, '${msg.author.id}');
+            INSERT INTO mod_log (timestamp, moderator, action, length_of_time, reason) VALUES
+            ('${date}', '${msg.author.id}', '${action}', 'N/A', '${reason}')`;
```

Comment:
Same here, I think we changed from 'N/A' to NULL. This is in all the files and I don't want to flood it with comments so I won't add any more of this specific comment.

Reply:
Done